debian/libmirclient11.install

```diff
@@ -0,0 +1 @@
+usr/lib/*/libmirclient.so.11
```

Comment:
Why is there still a libmirclient library?

Reply:
Because parts of the event and input code are there for historical reasons. I didn't want to mix moving it with what is mostly pure deletion. Also, we need to agree somewhere for it to go: mirplatform and mircommon are both possibilities.